modules/config/src/test/resources/application.conf

```diff
@@ -0,0 +1 @@
+rpc.server.port = 51111
```

Comment:
where is this used @cb372 ?

Reply:
In this test: https://github.com/higherkindness/mu-scala/pull/1463/files#diff-0ad51a2d7069328bafc89a9c1cfc566aee3f1e30a8d3a55be6912b17f246af39R32-R40